test/artifact-utils.spec.ts

```diff
@@ -66,6 +66,21 @@ describe('artifact-utils', () => {
       ).toMatchInlineSnapshot(`"https://mirror.example.com/v6.0.0/electron-v6.0.0-linux-x64.zip"`);
     });
 
+    it('should allow for setting only a base url when mirrorOptions.baseOnly is set', () => {
+      expect(
+        getArtifactRemoteURL({
+          arch: 'x64',
+          artifactName: 'electron',
+          mirrorOptions: {
+            mirror: 'https://mirror.example.com',
+            baseOnly: true,
+          },
```

Comment:
Add customDir` and `customFilename` here to show that they don't factor into the URL in this case?

Reply:
@malept it's a little weird to me that `nightly_mirror` isn't camelCase - is that intentional or would you be open to my changing that in a follow-up PR?

Reply:
I don't think it was intentional although you'd have to doublecheck with @MarshallOfSound 😁 

I'm OK with a PR to deprecate with a camel cased version.